src/app/ui/form/Form.tsx

```diff
@@ -33,7 +33,11 @@ const Form: FunctionComponent<FormProps> = ({
 
         if (erroredFormField) {
             erroredFormField.focus({preventScroll: true});
-            erroredFormField.offsetParent?.scrollIntoView({behavior: 'smooth', block: 'center', inline: 'center'});
+            try {
+                erroredFormField.offsetParent?.scrollIntoView({ behavior: 'smooth', block: 'center', inline: 'center'});
+            } catch {
+                erroredFormField.offsetParent?.scrollIntoView();
```

Comment:
Thanks for quick response. To me, using try/catch and then reverting to the supported way seemed to be a reasonable solution.